app.js

```diff
@@ -31,7 +30,40 @@ app.use(require('node-sass-middleware')({
 }));
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/', routes);
+app.get('/', function(req, res) {
+  res.render('index', { title: 'Jeffrey and Elyse' });
+});
+
+app.get('/rsvp', function(req, res) {
+  res.render('rsvp', { title: 'RSVP to Jeffrey and Elyse' });
```

Comment:
"RSVP to Jeffrey and Elyse" sounds kind of awkward. "to Jeffrey and Elyse's wedding party"?